docs/sidebars.js

```diff
@@ -62,9 +62,9 @@ module.exports = {
           'api-reference/utils/constSelector',
           'api-reference/utils/errorSelector',
           'api-reference/utils/waitForAll',
-          'api-reference/utils/waitForAny',
-          'api-reference/utils/waitForNone',
           'api-reference/utils/noWait',
```

Comment:
The `waitFor*()` entries are more related and should be kept together.  But, should be fine to move `noWait()` above those three.